charts/platform/values.yaml

```diff
@@ -188,6 +188,13 @@ debug:
   # -- Optional. Enable debug mode.
   enabled: false
 
+# -- ClickHouse configuration for cluster support
+clickhouse:
+  # -- Whether ClickHouse is running in a clustered environment
+  isCluster: false
```

Comment:
Do you think we need the `isCluster`, could we just use the clusterName is not empty to imply it is a cluster?